chain-server/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::compile_protos("chain-server.proto")?;
```

Comment:
Conventionally, you'd want the proto files to be in a different directory, see e.g. https://github.com/hyperium/tonic/blob/master/examples/build.rs